### PR TITLE
GIVCAMP-242: Downgrade to match last working PR.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Fix Safari back button bug
- Reverts back to last known working versions

# Review By (Date)
- Whenever

# Criticality
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. In Safari navigate to [Preveiw Build](https://deploy-preview-177--giving-campaign.netlify.app/)
2. Scroll down and click on a page to navigate to a new route
3. Click on the back button
4. Wait for page to load then scroll the page to ensure route 'sticks'

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-20
```
Comment (sherakama) [inline .nvmrc:1]: This might be a contributor

File: package.json [hunk 2]
```diff
@@ -63,6 +63,6 @@
     "local-ssl-proxy": "^2.0.5"
   },
   "engines": {
-    "node": ">=20.0.0"
```
Comment (sherakama) [inline package.json:66]: Rolling back.

**Comment by yvonnetangsu** [comment]
YES! The back button worked on both desktop Safari and mobile browsers. You made my day @sherakama 🚀 😄 Thank you for figuring this out.

Some errors with the newer stories however - I suspect that's not caused by the node downgrade but maybe the revalidate 😂 
https://deploy-preview-177--giving-campaign.netlify.app/stories/cardinal-rules
https://deploy-preview-177--giving-campaign.netlify.app/stories/using-machine-learning-to-help-refugees-succeed
https://deploy-preview-177--giving-campaign.netlify.app/stories/from-human-health-to-climate-health

**Comment by sherakama** [comment]
Ok, I'll remove the revalidate and troubleshoot that in the other PR.

**Comment by yvonnetangsu** [comment]
I cleared cache and rebuilt on Netlify - there's some build error that's caused by me changing that caption field to RichText - should be if I merge my branch into your branch. I'm going to get that one done first. Thanks @sherakama !

**Comment by sherakama** [comment]
Hmm if it was the revalidate that fixed the back button issue I am going to close this.

**Comment by yvonnetangsu** [comment]
This issue is so mysterious 😂 Thank you for your help @sherakama - very much appreciate it. Have a great weekend!

**Pull request closed without merging**